live/root/etc/systemd/system/agama-password-dialog.service

```diff
@@ -0,0 +1,45 @@
+[Unit]
+Description=Interactively set the Agama/root password in a dialog
+
+# before starting the SSH and Agama server so they use the new password
+Before=sshd.service
+Before=agama-web-server.service
```

Comment:
np: I would try to be consistent with the style in `agama-password-cmdline.service` and use:

```
Before=sshd.service agama-web-server.service
```

or

```
Before=sshd.service
Before=agama-web-server.service
```

Whatever you prefer, but the same.